model_compression_toolkit/core/common/framework_implementation.py

```diff
@@ -75,12 +75,16 @@ def to_tensor(self, tensor: np.ndarray) -> Any:
     @abstractmethod
     def model_reader(self,
                      model: Any,
-                     representative_data_gen: Callable) -> Graph:
+                     representative_data_gen: Callable,
+                     model_leaf_layers: list = None) -> Graph:
         """
         Convert a framework's model into a graph.
         Args:
             model: Framework's model.
             representative_data_gen (Callable): Dataset used for calibration.
+            model_leaf_layers (list): For PyTorch only!
+            List of the module's custom layers, these layers shouldn't be divided into
+            their submodules and their quantization will not be optimized.
```

Comment:
List of the module's custom layers. These layers shouldn't be divided into their submodules. Please note that their quantization will not be optimized.